runtime/vm/BytecodeInterpreterFull.cpp

```diff
@@ -0,0 +1,29 @@
+/*******************************************************************************
+ * Copyright (c) 2020, 2020 IBM Corp. and others
+ *
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+ * or the Apache License, Version 2.0 which accompanies this distribution and
+ * is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * This Source Code may also be made available under the following
+ * Secondary Licenses when the conditions for such availability set
+ * forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+ * General Public License, version 2 with the GNU Classpath
+ * Exception [1] and GNU General Public License, version 2 with the
+ * OpenJDK Assembly Exception [2].
+ *
+ * [1] https://www.gnu.org/software/classpath/license.html
+ * [2] http://openjdk.java.net/legal/assembly-exception.html
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+ *******************************************************************************/
+
+#include "j9cfg.h"
+
+#if defined(OMR_GC_FULL_POINTERS)
+#define J9_OVERRIDE_COMPRESS_OBJECT_REFERENCES FALSE
+#define FULL_VERSION
```

Comment:
Rather than doing this, I think it would be better to hoist the class/loop names out into the wrapper .cpp files.

Reply:
I will move `#define LOOP_NAME bytecodeLoopFull` etc to be declared per wrapper file. Each wrapper file will still include BytecodeInterpreter.inc, and the rest of the .inc will remain the same?

Reply:
Sounds good - please do the same for the debug interpreter (though you will still need to define DEBUG_VERSION for that case).